server/src/main/java/org/apache/iotdb/db/conf/IoTDBConfig.java

```diff
@@ -760,6 +767,12 @@
    */
   private boolean enableRpcService = true;
 
+  /**
+   * whether enable the influxdb rpc service. This parameter has no a corresponding field in the
+   * iotdb-engine.properties
+   */
+  private boolean enableInfluxDBRpcService = true;
```

Comment:
```suggestion
  private boolean enableInfluxDBRpcService = false;
```

Set the property false by default.